opm/material/densead/Evaluation.hpp

```diff
@@ -239,11 +249,8 @@ class Evaluation
     Evaluation& operator/=(const RhsValueType& other)
     {
         // values and derivatives are divided
-        ValueType factor = (1.0/other);
-        for (unsigned idx = 0; idx < length_; ++idx)
-            data_[idx] *= factor;
-
-        return *this;
+        const ValueType factor = (1.0/other);
+        return this->operator *=( factor );
```

Comment:
can you remove the whitespace in `operator*()`. At first, I was a bit confused by this.